scss/_patterns_tabs.scss

```
@@ -93,4 +93,37 @@
       }
     }
   }
+
+  .p-tabs-segmented {
+    &.is-dense .p-tabs-segmented__button {
+      @extend %vf-button-dense-vertical-spacing;
+    }
+
+    .p-tabs-segmented__list {
+      display: flex;
```

without this, the buttons are stretched to fill the parent. If for whatever reason something is different (e.g. if you apply `.is-dense` on one of the buttons, not that you should) then unexpected stuff happens.
```suggestion
      display: flex;
      align-items: flex-start;
```